data/free worlds reconciliation.txt

```diff
@@ -1554,6 +1558,610 @@ mission "FW Pug 3C"
+			`	"Do you know how the Pug cut you off?"`
+			`	"Look, I'm no scientist, and I have no idea how Hyperspace links work. Maybe someone on the other end can help you."`
+				decline
+
+
+mission "FW Pug 1 Know Return"
+	landing
+	to offer
+		has "FW Pug 1 Know: active"
+		has "event: pug invasion 4"
+	
+	on offer
+		conversation
+			`You receive an urgent message from JJ: "Captain <last>, return to New Tibet at once! We've received word that the Syndicate is somehow severing their hyperspace connections to the rest of human space. We're calling an emergency council meeting, to convene as soon as you arrive here."`
+				decline
+
+
+
+mission "FW Pug 2 Know"
+	landing
+	name "Meet with Parliament"
+	description "Travel with Alondo to Earth to meet with Parliament. They are now much more willing to meet with you given how real the threat seems."
+	autosave
+	source "New Tibet"
+	destination "Earth"
+	to offer
+		has "FW Pug 1 Know: done"
+	passengers 1
+	blocked `As you are landing, Alondo contacts you and says, "Do you have a bunk free for me?" You do not, so you will need to free one up in order to continue this mission.`
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	
+	on offer
+		conversation
+			`By the time you arrive, Alondo and Freya have joined JJ and Katya on New Tibet; the entire Council is present. "Here's the story," says JJ. "A few weeks ago, hyperspace links on the edge of Syndicate space began collapsing. We weren't sure why."`
+			`	"I know why." you interrupt. The silence this statment leaves behind is deafening.`
+			`	Evetuly Katya asks "<first>, is it the Syndicate?"`
+			`	"No, it is an alien race called the Pug. Thay say thay are there to promote peace, but I don't know their true intentions."`
+			`	JJ says, "The Senate had agreed that we must put our differences with the Republic aside, and meet with Parliament to deal with this crisis. But that was before you told us what has really happened."`
+			`	"Where did the Pug come from? And are they attacking Human ships?" Katya asks starting to sound worried.`
+			`	"They came from a system called Deneb, the star out beyond Zeta Aquilae. They didn't attack my ship when I arrived and nothing on the ground looked like it had been damaged." At this Katya looks relived. "There are some people who are resisting the Pug but there wasn't a single Free Worlds ship in orbit or on the ground so they were probably destroyed when the Pug invaded."`
+			`	Alondo contemplates this before speaking up "We can easily stop are own people panicking but we can't use the Net to tell the Republic, they just won't believe it."`
```

Comment:
Line 1702, dialogue should read "We can easily stop our own people..." instead of "stop are own people..."